LocalPackages/tunnelkit/Sources/TunnelKitOpenVPNAppExtension/OpenVPNTunnelProvider.swift

```diff
@@ -321,12 +323,20 @@ open class OpenVPNTunnelProvider: NEPacketTunnelProvider {
     // MARK: Wake/Sleep (debugging placeholders)
 
     open override func wake() {
-        log.verbose("Wake signal received")
+        log.info("Wake signal received")
```

Comment:
We don't log at both. It was previously logged at verbose level. I changed it to log at info level, so that it appears in our log (we use debug log level, which shows info() messages as well).